src/components/Display.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+
+export default function Display({ result }) {
+  return (
+    <div className="">
```

Comment:
Please leave out the class name attribute if you don't intend to include classes in your component. This will make your codebase smaller in the long run and less redundant.

Reply:
Done